.github/pull_request_template.md

```diff
@@ -33,18 +33,24 @@ Addresses #*PR# HERE*
 - [ ] If there are any front-end changes, before/after screenshots are included
 - [ ] Critical and brittle code paths are covered by unit tests
 - [ ] The change has been added to the `changelog`
-
-## Updating KDS in Kolibri
-- [ ] After this PR has been merged into KDS, a separate PR has been submitted to point the Kolibri `develop` branch to the most recent version of KDS
+- [ ] A separate PR has been submitted to point the `unstable` branch of the affected Kolibri product to the most recent version of KDS (see post-merge updates for details)
```

Comment:
Is this item intended to be checked before or after the post-merge updates? It seems to describe similar tasks as

```
- [ ] Learning Platform update PR is submitted
- [ ] Learning Platform update PR is merged
```

Reply:
Ah yes. Let's get rid of this.